airflow/debug.py

```diff
@@ -0,0 +1,8 @@
+import os
```

Comment:
note to self to write more "developing" documentation

Reply:
Added some notes in the docs issue for some useful and quick screencasts--

https://github.com/cal-itp/data-infra/issues/83#issuecomment-855996337